### PR TITLE
feature/api with authServices example

### PR DESCRIPTION
**Pull request by Brix101**

### CODE REVIEW

File: src/app/store.js [hunk 1]
```diff
@@ -33,6 +35,7 @@ export const store = configureStore({
           REGISTER,
         ],
       },
-    }).concat(logger),
-  // .concat(api.middleware),
+    })
+      .concat(api.middleware)
+      .concat(logger)
```
Comment (Brix101) [inline src/app/store.js:40]: disable logger on production

**Review by garkgodwin — APPROVED** [review]
reviewed and approved

**Review by garkgodwin — APPROVED** [review]
Reviewed and approved

**Review by garkgodwin — APPROVED** [review]
Reviewed and approved

**Review by garkgodwin — APPROVED** [review]
Reviewed and approved

**Comment by garkgodwin** [comment]
noice

File: src/app/store.js (hunk 1 above)
Comment (garkgodwin) [reply src/app/store.js:40]: ok noted

**Pull request merged**